crates/ibc/src/clients/ics07_tendermint/client_state.rs

```diff
@@ -262,7 +264,7 @@ impl Ics2ClientState for ClientState {
     }
 
     fn client_type(&self) -> ClientType {
-        ClientType::Tendermint
+        ClientType::new(TENDERMINT_CLIENT_TYPE)
```

Comment:
It might be a little bit nicer to have a `const fn ClientType::tendermint() -> ClientType` constructor rather than reference the constant directly.

Reply:
tendermint: [26c6c99](https://github.com/cosmos/ibc-rs/pull/189/commits/26c6c992213325081b92bd329037526cf1011497)

mock: [676f38e](https://github.com/cosmos/ibc-rs/pull/189/commits/676f38ee8bdb5b6c9ef5366031c66051db69a155)